sdc/tests/tests_perf/generator.py

```diff
@@ -22,7 +22,8 @@ def gen(cases, method, class_add, typ, prefix=''):
 def test_gen(name, params, data_length, prefix, call_expression):
     func_name = 'func'
     if call_expression is None:
-        call_expression = '{}{}({})'.format(prefix, name, params)
+        parts = [prefix, '{}({})'.format(name, params)]
+        call_expression = '.'.join(parts)
```

Comment:
In case of `len(parts) == 2`, maybe better to use formatted string as it was before, but inserting dot to there?